services/productService.js

```diff
@@ -0,0 +1,22 @@
+const Product = require('../models/productModel');
+
+const addProduct = async (data) => {
+  const product = new Product(data);
+  await product.save();
+};
+
+const allProducts = async () => {
+  const products = await Product.find();
+  return products;
```

Comment:
in javascript when we function that contains onlu online we can omit return line and asyn await keywords so we can do it like this :
 const allProducts=()=>Product.find()